docs/1.0. Certificate Provisioning.md

```diff
@@ -329,6 +329,7 @@ An EST Client SHOULD periodically check the revocation status of both the Root C
 - The certificate returned by the EST Server MAY not be valid until sometime in the future
   - The 'Not Before' date MUST be checked before using the certificate
 - Care SHOULD be taken when implementing the certificate renewal process, to make sure the renewal does not affect user operation in a critical way (eg. loss of control by connected NMOS clients or affect the primary operation of the device).
+- If an EST Client cannot perform certificate renewal without affecting user operation in a critical way, then EST MAY be automatically disabled after successful certificate enrolments, preventing a renewal while the device is in use. System administrator MUST then monitor the certificate expiry and enable EST at a suitable time, forcing the EST Client to perform an enrolment.
```

Comment:
```suggestion
- If an EST Client cannot perform certificate renewal without affecting user operation in a critical way, then the device SHOULD provide a configuration option to disable EST automatically after an initial successful certificate enrolment, preventing a renewal while the device is in use. System administrators are strongly RECOMMENDED to monitor these devices for certificate expiry and enable EST manually at a suitable time, forcing the EST Client to perform a renewal.
```

Reply:
On the basis that devices which can auto-renew may still do so automatically, I'd suggest also adding the following as a separate note before the above bullet:

- EST Clients MUST provide a configuration option to enable and disable renewals such that potential service interruptions during critical periods can be prevented.